### PR TITLE
Dictionary cards should be the same across the application

### PR DESCRIPTION
**Pull request by muhozi**
# JIRA TICKET NAME:
[Dictionary cards should be the same across the application](https://issues.openmrs.org/browse/OCLOMRS-249)

# Summary:
The dictionary cards don't look the same for both homepage and dictionaries page, this has to be fixed. Regarding dictionary cards, divider line on home page that is in dictionaries section has to be aligned with the dictionary cards.

### CODE REVIEW

**Comment by muhozi** [comment]
@Rhotimee Yes, it is used.

**Comment by Rhotimee** [comment]
> Hey @Rhotimee Yes, it is used.

p-1, p-2, p-3 are bootstrap classes. I don't think it a good practice to style a bootstrap class. 
I think it'll be better if you use custom classname and style it with CSS here.

**Comment by muhozi** [comment]
> > Hey @Rhotimee Yes, it is used.
> 
> p-1, p-2, p-3 are bootstrap classes. I don't think it a good practice to style a bootstrap class.
> I think it'll be better if you use custom classname and style it with CSS here.

@Rhotimee To remove them or change those class names was not the scope of the ticket. Yeah, you're right. It would be better to use the custom class names, I think that we should create a ticket for this.

**Comment by Rhotimee** [comment]
> > > Hey @Rhotimee Yes, it is used.
> > 
> > 
> > p-1, p-2, p-3 are bootstrap classes. I don't think it a good practice to style a bootstrap class.
> > I think it'll be better if you use custom classname and style it with CSS here.
> 
> @Rhotimee To remove them or change those class names was not the scope of the ticket. Yeah, you're right. It would be better to use the custom class names, I think that we should create a ticket for this.

Ok. Cool. It'll be nice if this can be addressed in another ticket. 

LGTM

File: src/components/userDasboard/container/UserDashboard.jsx [hunk 1]
```diff
@@ -43,7 +43,7 @@ export class UserDashboard extends Component {
 
   gotoDictionary = (url) => {
     this.props.history.push(url);
-  }
```
Comment (dondrzzy) [inline src/components/userDasboard/container/UserDashboard.jsx]: @muhozi, Is this a formatting change?

Comment (muhozi) [reply src/components/userDasboard/container/UserDashboard.jsx]: @dondrzzy Thanks for pointing this out. This is not related to the ticket. I reverted this change.

**Pull request merged**